src/aap_eda/analytics/analytics_collectors.py

```diff
@@ -0,0 +1,348 @@
+import os
+import platform
+from datetime import datetime
+
+import distro
+from ansible_base.resource_registry.models.service_identifier import service_id
+from django.conf import settings
+from django.db.models import Manager, Q
+from insights_analytics_collector import CsvFileSplitter, register
+
+from aap_eda.analytics.collector import AnalyticsCollector
+from aap_eda.core import models
+from aap_eda.utils import get_eda_version
+
+
+@register(
+    "config",
+    "1.0",
+    description="General platform configuration.",
+    config=True,
+)
+def config(**kwargs) -> dict:
+    install_type = "traditional"
+    if os.environ.get("container") == "oci":
+        install_type = "openshift"
+    elif "KUBERNETES_SERVICE_PORT" in os.environ:
+        install_type = "k8s"
```

Comment:
What about a VM install type?

Reply:
> What about a VM install type?

Per the [discussion in slack](https://redhat-internal.slack.com/archives/C07KQJ2C5KM/p1725464425587529) , VM install type is the `traditional`